app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] Good job working on this, but please kindly ensure there is a `new` method for the `CommentsController`